src/main/java/wooteco/subway/ui/LineController.java

```diff
@@ -0,0 +1,51 @@
+package wooteco.subway.ui;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.service.LineService;
+
+import java.net.URI;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineService service;
+
+    public LineController(LineService service) {
+        this.service = service;
+    }
+
+    @PostMapping()
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = service.save(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<LineResponse>> showLines() {
+        return ResponseEntity.ok().body(service.findAll());
+    }
+
+    @GetMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+        return ResponseEntity.ok().body(service.findById(id));
+    }
+
+    @DeleteMapping(value = "/{id}")
+    public ResponseEntity<LineResponse> deleteLine(@PathVariable Long id) {
+        service.deleteById(id);
+        return ResponseEntity.noContent().build();
+    }
+
+    @PutMapping(value = "/{id}")
+    public ResponseEntity<LineResponse> updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
+        service.update(id, lineRequest);
+        return ResponseEntity.ok().build();
+    }
```

Comment:
```suggestion
    public ResponseEntity<Void> updateLine(@PathVariable Long id, @RequestBody LineRequest lineRequest) {
        service.update(id, lineRequest);
        return ResponseEntity.ok().build();
    }
```
응답 body가 없다면 `Void`를 활용해주세요!
위의 코멘트를 활용하면 리턴값이 없어도 될 것 같네요 :)

Reply:
리턴값이 없도록 수정했습니다 :)